agent/config/config.go

```diff
@@ -80,6 +80,9 @@ const (
 	// minimumNumImagesToDeletePerCycle specifies the minimum number of images that to be deleted when
 	// performing image cleanup.
 	minimumNumImagesToDeletePerCycle = 1
+
+	// Default cgroup prefix for ECS tasks
+	DefaultTaskCgroupPrefix = "/ecs"
```

Comment:
I think @richardpen's asking if we should let it be configurable, which seems like a reasonable thing to do from an extensibility perspective.